docs/source/references/schema.json

```diff
@@ -903,7 +903,7 @@
       },
       {
         "name": "transactions_hash",
-        "type": "HashOf<MerkleTree<SignedTransaction>>"
+        "type": "Option<HashOf<MerkleTree<SignedTransaction>>>"
```

Comment:
Breaks ABI

Reply:
This cannot be avoided to fix the bug needed for backport. And it doesn't have to break abi because as far as I remember we have some niche optimization in place for hashes

Reply:
This is required for the main branch, but for now, we could revert it to keep the block explorer functional on the testnet